src/sqlancer/duckdb/DuckDBSchema.java

```diff
@@ -41,9 +41,19 @@ public static class DuckDBCompositeDataType {
 
         private final int size;
 
+        // This is used to handle the type that out of the scope of our code
```

Comment:
Hmm, I don't understand the comment and use of this. Could we perhaps explain or give an example?